spec/requests/partners_requests_spec.rb

```diff
@@ -297,6 +297,7 @@
         subject.call
         expect(UserInviteService).to have_received(:invite).with(
           email: email,
+          name: partner.name,
```

Comment:
Switch this to use the new generated `name`
```suggestion
          name: name,
```